mscore/musescore.cpp

```diff
@@ -6992,7 +6994,7 @@ bool MuseScore::saveMp3(Score* score, QIODevice* device, bool& wasCanceled)
                                           continue;
                                     e.setChannel(a->channel());
                                     int syntiIdx= synth->index(score->masterScore()->midiMapping(a->channel())->articulation()->synti());
-									synth->play(e, syntiIdx);
+                                    synth->play(e, syntiIdx);
```

Comment:
this one was particularly bad before your change, so yes, keep it, even if entirely unrelated to your PR